dbt/adapters/databricks/connections.py

```diff
@@ -475,6 +476,17 @@ class DatabricksConnectionManager(SparkConnectionManager):
     TYPE: str = "databricks"
     credentials_provider: Optional[TCredentialProvider] = None
 
+    def cancel_open(self) -> List[str]:
+        from dbt.adapters.databricks.python_submissions import BaseDatabricksHelper
```

Comment:
import at the top please.  We only import in place like this if the thing we're importing is too heavy to do at start up.

Reply:
done!